api/src/server.ts

```diff
@@ -148,6 +148,17 @@ friends(app);
 canny(app);
 user(app);
 
+// Serve public catalog files without authentication
+app.use(
+  '/api/static/catalogs/public',
```

Comment:
@bearsyankees I think the issue is that the `/api/static/catalogs` is a parent path of this, so if the former requires auth, this will too. I suggest moving this to something like `/api/static-public`, or you would have to move the currently needing-auth data to a subdirectory.

Also, I suggest reusing the data as much as possible, and only merge the extra data, instead of using an entirely different set of data.

Reply:
No this works for auth. So you don't think having just 2 jsons is the way to go? don't think it takes up that much space but i hear u def could be better to always pull from public except for eval data... imma try and get this working first then figure out how to optimize